BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -504,4 +520,19 @@
         ]]>
     </string>
 
+    <!-- Voice Note Editor -->
+    <string name="voice_note_name">Sprachnotiz vom </string>
+    <string name="default_played_timer">00:00</string>
+    <string name="record_permission_needed">Erlaubnis zur Aufnahme erforderlich</string>
+    <string name="record_permission_alert_msg">
+        Hey, wir brauchen zuerst Zugriff auf das Mikrofon - bitte erlauben und nochmal versuchen!
+    </string>
+    <string name="voice_note_saved">Die Sprachnotiz wurde gespeichert.</string>
+
+    <!-- Voice Note Editor Image Descriptions -->
+    <string name="record_microphone">Record Microphone Icon</string>
+    <string name="stop_button_note">Stop Button</string>
+    <string name="play_button_note">Play Button</string>
+    <string name="microphone">Microphone Icon</string>
```

Comment:
What are these? If these are sole descriptions for icons, please move them up in the respective section and rename them accordingly. Also, please take some time to add more detail to your description for visually impaired.

Please also translate to German :)

Reply:
You are right - I moved them to the general Image Descriptions section and rewrote the content to be more specific and translated it to german.

Reply:
Great, thanks!